config/locales/ar.yml

```diff
@@ -103,6 +103,8 @@ ar:
   components:
     figure:
       image_credit: 'حقوق ملكية الصورة: %{credit}'
+    print_link:
```

Comment:
Hmm, this commit has all the locale changes (rather than separate commits for the component locales and the travel advice locales, which we have with the other move PRs)

Reply:
Ah, are there no translations for the actual component that got moved?

Reply:
I see, the component is just a wrapper around an SVG. Okay, no problem then.